papaya-web/src/main/java/pl/edu/agh/papaya/rest/projects/service/SprintsRestService.java

```diff
@@ -125,10 +129,33 @@
                 .map(timeBurned -> (double) sprint.getTimePlanned().toMinutes() / timeBurned.toMinutes())
                 .orElse(0.d);
 
-        return ResponseEntity.ok(new SprintSummaryDto().membersAvailability(userAvailabilityDtos)
+        return ResponseEntity.ok(createSprintSummaryDto(userAvailabilityDtos, totalAvailableTime, coefficient));
+    }
+
+    private SprintSummaryDto createSprintSummaryDto(List<UserAvailabilityDto> userAvailabilityDtos,
+            Duration totalAvailableTime, double coefficient) {
+        return new SprintSummaryDto().membersAvailability(userAvailabilityDtos)
                 .totalAvailableTime(totalAvailableTime.toMinutes())
-                .sprintCoefficient(coefficient)
-        );
+                .sprintCoefficient(coefficient);
+    }
+
+    private List<UserAvailabilityDto> getUserAvailabilityDtos(Project project, List<Availability> availabilities) {
+        List<UserAvailabilityDto> usersWithNoAvailabilities = getUsersWithNoAvailabilities(project, availabilities);
+        List<UserAvailabilityDto> usersWithAvailabilities = userAvailabilityMapper.mapToApi(availabilities);
+        return Stream.concat(usersWithAvailabilities.stream(), usersWithNoAvailabilities.stream())
+                .collect(Collectors.toList());
+    }
+
+    private List<UserAvailabilityDto> getUsersWithNoAvailabilities(Project project, List<Availability> availabilities) {
+        List<User> users = project.getUsersInProject()
+                .stream()
```

Comment:
W sumie chyba wszystko okej, ale żeby nie było że nie przeczytałem, czy coś, to nie podoba mi się jak `stream()` jest w innej linijce niż obiekt na którym jest wywoływane xd

Reply:
Proszę bardzo